daemon/daemon.go

```diff
@@ -169,19 +170,20 @@ func (d *Daemon) Run() error {
 	}
 	d.containerMgr = containerMgr
 
+	if err := containerMgr.Restore(ctx); err != nil {
+		return err
+	}
+
+	// ensure all containers have been restored
+	time.Sleep(100 * time.Millisecond)
+
 	networkMgr, err := internal.GenNetworkMgr(d.config, d)
 	if err != nil {
 		return err
 	}
 	d.networkMgr = networkMgr
 	containerMgr.(*mgr.ContainerManager).NetworkMgr = networkMgr
 
-	// Notes(ziren): we must call containerMgr.Restore after NetworkMgr initialized,
-	// otherwize will panic
```

Comment:
If change this, won't panic?

Reply:
if the NetworkManange is nil, markStoppedAndRelease will return directly. So it won't panic